src/main/java/UseCase/editPlaylist.java

```diff
@@ -20,6 +22,7 @@ public String removeSong(editPlaylistInputData inputData) {
         }
         return inputData.getSong().getName()+" removed ";
     }
+
     // TODO: Work on view later
     public String addSong(editPlaylistInputData inputData){
```

Comment:
Instead of returning a String directly, you'd have passed in a Presenter in the initializer whose method you'd call

Reply:
This is true, I havent coded that part as I was more focused on correctly coding the create playlist use case, then updating edit playlist to do the same